keyform/static/keyform/js/base.js

```diff
@@ -0,0 +1,44 @@
+$(document).ajaxSend(function(event, xhr, settings)
+{
+    function getCookie(name)
```

Comment:
This function looks similar to [this](https://docs.djangoproject.com/en/1.11/ref/csrf/#acquiring-the-token-if-csrf-use-sessions-is-false). If you got the code from there we may want to give them credit.